buildkitd/entrypoint.sh

```diff
@@ -34,6 +34,23 @@ if [ -n "$GIT_URL_INSTEAD_OF" ]; then
 
 fi
 
+# setup git credentials and config
+i=0
+while true
+do
+    varname=GIT_CREDENTIALS_"$i"
+    eval data=\$$varname
+    if [ -n "$data" ]
+    then
+        echo $data | base64 -d > /usr/bin/git_credentials_"$i"
```

Comment:
Hmm.. we shouldn't write these to disk. Can we just keep them as env vars and never write them as files in the clear?

Reply:
I changed the helper to echo out (and decode) it from the env now.